libs/service-portal/core/src/lib/navigation/masterNavigation.ts

```diff
@@ -332,6 +332,26 @@ export const servicePortalMasterNavigation: ServicePortalNavigationItem[] = [
         },
         description: m.accessControlDescription,
       },
+      // Aðgangsstýring umboð
+      {
+        name: m.accessControl,
```

Comment:
Maybe add something like "V2" to this label for now while users might see both at the same time? :)